resource/report.go

```diff
@@ -0,0 +1,28 @@
+package resource
+
+import (
+	. "github.com/talbright/go-desk/types"
+)
+
+
+type InsightsV3Report struct {
+	Request *InsightsV3Query `json:"request"`
+	Header  []string         `json:"header"`
+	Data    [][]interface{}  `json:"data"`
+}
+
+func (r *InsightsV3Report) GetData() []map[string] interface{} {
```

Comment:
Can you add a unit test for this (```report_test.go``)? Thus far I've resisted against writing unit tests that simply exercise the struct json mappings (though maybe it would have caught #16 that you fixed). But for cases where a resource does provide utility functions, such as https://github.com/talbright/go-desk/blob/60537dd851062dd90102e9c4adc6c19a7b38c8df/resource/customer_test.go, I think it's a good idea to provide test coverage.